src/routes/user/register/1.svelte

```diff
@@ -0,0 +1,384 @@
+<script lang="ts">
+	import Markdown from '$components/markdown.svelte';
+	import * as _ from 'lodash-es';
+	import { z } from 'zod';
+	import { handle_input } from '$functions/forms/handle_input';
+	import { zxcvbn, zxcvbnOptions, type OptionsType } from '@zxcvbn-ts/core';
+	import { cn } from '$functions/classnames';
+	import { autofocus } from '$functions/forms/autofocus';
+	import Arrow from '$icons/shapes/arrow.svelte';
+	import Info from '$icons/shapes/info.svelte';
+	import Tick from '$icons/shapes/tick.svelte';
+	import ArrowUpRight from '$icons/shapes/arrow_up_right.svelte';
+	import CoreText from '$icons/text/core.svelte';
+	import type { FormFieldState, PageProps } from './types';
+
+	let { on_submit, pages_state }: PageProps = $props();
+
+	// let pages_state = new Array({});
+
+	let combined_state: { [key: string]: string };
+
+	$effect.pre(() => {
+		$inspect(pages_state);
```

Comment:
remove this

Reply:
States are not working atm, will remove all consoles after testing :+1: